README.md

```diff
@@ -10,11 +10,12 @@ Moneo orchestrates metric collection (DCGMI + Prometheus DB) and visualization (
 
 There three categories of metrics that Moneo monitors:
 1.	Device Counters
```

Comment:
should we say GPU Counters here to avoid confusion?

Reply:
I can fix the docs to say that.

Reply:
Addressed